targets/TARGET_STM/TARGET_STM32L4/TARGET_STM32L475xG/TARGET_DISCO_L475VG_IOT01A/PeripheralPinMaps.h

```diff
@@ -52,6 +52,94 @@
 #include "PinNamesTypes.h"
 #include <mstd_cstddef>
 
+//*** GPIO ***
+
+MSTD_CONSTEXPR_OBJ_11 PinMap PinMap_GPIO[] = {
```

Comment:
If this is only for testing I would that this code only exists for tests ... as of now this table seems to be unconditionally compiled.  Also we want to be able to test any platform with FPGA test shield and I would like to avoid that we have to add such a dummy table to all our Targets  ... all in all I am not in favor of this change, I hope that can be a better alternative

Reply:
I've just confirmed that the `PinMap_GPIO` symbol is discarded at link time if not used (by building a blinky example). When building `tests-mbed_hal_fpga_ci_test_shield-gpio` the elf does contain it, as expected.
There is no need to add this to other targets if they pass the FPGA GPIO test. A default, weak implementation is used (https://github.com/Armmbed/mbed-os/blob/master/hal/mbed_gpio.c#L77). We're overriding the `PinMap_GPIO` only if there is a HW limitation (like a pull-up) for any pins.